lightly/api/patch.py

```diff
@@ -0,0 +1,130 @@
+import logging
+from typing import Any, Dict, Type
+
+
+def rest_client_flatten_array_query_parameters(rest_client_cls: Type):
```

Comment:
Moved this method from `patch_rest_client.py` and renamed the `rest_client` argument to `rest_client_cls`